src/grpc/grpc_server.h

```diff
@@ -113,10 +101,17 @@ class Server {
       const std::shared_ptr<SharedMemoryManager>& shm_manager,
       const Options& server_options, std::unique_ptr<Server>* server);
 
-  static bool CreateWrapper(
+  static TRITONSERVER_Error* Create(
       std::shared_ptr<TRITONSERVER_Server>& server, UnorderedMapType& data,
-      std::unique_ptr<Server>* service,
-      const RestrictedFeatures& restricted_features);
+      const RestrictedFeatures& restricted_features,
+      std::unique_ptr<Server>* service);
+
+  static void getSocketOptions(SocketOptions& options, UnorderedMapType& data);
+  static void getSslOptions(SslOptions& options, UnorderedMapType& data);
+  static void getKeepAliveOptions(
+      KeepAliveOptions& options, UnorderedMapType& data);
+
+  static void getOptions(Options& options, UnorderedMapType& data);
```

Comment:
Always start uppercase for style consistency, `Get`

Reply:
Changed to `Get*()`.